src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const stringArray = sourceString.split(';');
+  const filterArray = stringArray.filter((item) => item.trim().length);
+
+  for (const words of filterArray) {
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. You can use `forEach` method instead of `for...of` loop.